tests/default_vars.sh

```diff
@@ -42,6 +61,12 @@ elif [[ $MACHINE_ID = wcoss_dell_p3 ]]; then
   TASKS_cpl_bmrk=560; TPN_cpl_bmrk=28; INPES_cpl_bmrk=6; JNPES_cpl_bmrk=8
   THRD_cpl_bmrk=1; WPG_cpl_bmrk=24; MPB_cpl_bmrk="0 287"; APB_cpl_bmrk="0 311"
   OPB_cpl_bmrk="312 431"; IPB_cpl_bmrk="432 479"; WPB_cpl_bmrk="480 559"
+  NPROC_ICE_cpl_bmrk=48
+
+  TASKS_cpl_bmrk_mpi=600; TPN_cpl_bmrk_mpi=40; INPES_cpl_bmrk_mpi=6; JNPES_cpl_bmrk_mpi=8
```

Comment:
The cpl_bmrk_mpi test only has wave tasks changed. I am wondering which test, cpld_bmark_p7 or cpld_bmark_mpi_p7, has better load balance?

Reply:
In the end, I was simplifying the changes to narrow them to just WW3 --- as that was the focus of the debug.  If this test is left in, I can change other pet tasks if desired. 

I did not do any load balancing on the cpl benchmark test as the wave grid did not change.  I assumed it was load-balanced as is and the other was just for testing the wave MPI issues.